pkg/rancher-components/src/components/Form/Radio/RadioGroup.vue

```diff
@@ -199,42 +237,42 @@ export default defineComponent({
 </script>
 
 <template>
-  <fieldset>
+  <div>
```

Comment:
Why are we removing the fieldset here?

Reply:
We were "bloating" the HTML when all we required was the proper wiring and passing of those attributes.